vue-reviews/src/data/firebase-config.js

```diff
@@ -1,14 +1,14 @@
 import firebase from 'firebase';
 
 var firebaseConfig = {
-    apiKey: "AIzaSyDgQjpMQf2p421ymTNbY-ZBf_giRphPUFw",
-    authDomain: "codereview-vue.firebaseapp.com",
-    databaseURL: "codereview-vue.firebaseio.com",
-    messagingSenderId: "463670284887",
-    senderId : "463670284887"
+  apiKey: "AIzaSyBXGbcm4vxcD4sKLcyOqBLl__O8Q4fPYdE",
+  authDomain: "reviewappforira.firebaseapp.com",
+  databaseURL: "reviewappforira.firebaseio.com",
+  messagingSenderId: "463670284887",
+  senderId: "463670284887"
```

Comment:
Please try using git ignore with "firebase-config.js", no need to import new values from test database here.